btr-web/btr-main-app/components/individual-person/tax-info/TaxNumber.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div class="flex flex-col py-5">
+    <div class="flex items-center mb-2 py-1">
+      <URadio
+        id="taxNumberRadioButton"
+        v-model="selectedButton"
+        :value="`CRA Tax Number`"
+        @change="handleRadioButtonChange(`CRA Tax Number`)"
+      />
+      <UFormGroup :name="name" class="ml-5">
+        <UInput
+          v-model="taxNumber"
+          type="text"
+          :disabled="selectedButton !== 'CRA Tax Number'"
+          variant="bcGov"
+          :placeholder="$t('placeholders.taxNumber')"
+          class="w-80"
+          @blur="formatInput"
+        />
+      </UFormGroup>
+    </div>
+    <div class="flex items-center mb-2 py-2">
+      <URadio
+        id="noTaxNumberRadioButton"
+        v-model="selectedButton"
+        :value="`No Tax Number`"
+        @change="handleRadioButtonChange(`No Tax Number`)"
+      />
+      <label class="ml-5"> This person does not have a CRA Tax Number </label>
```

Comment:
this should be in lang file. Actually all new texts we are adding should be in en.json in lang dir.